machine_learning_hep/data/data_prod_20200304/database_ml_parameters_Dspp.yml

```diff
@@ -295,6 +295,7 @@ Dspp:
         data: "trigger_hasbit_INT7==1"
         mc: null
         weighttrig: false
+        usetriggcorrfunc: true
```

Comment:
I guess we need to add this in the other DBs with the value `True`, right @arthur-gal @DelloStritto @ginnocen ? Just saw that it is taken in the `AnalyzerDhadrons_mult` without checking whether the key exists. So either we make it optional or add the key everywhere.